helm/app-operator/templates/deployment.yaml

```
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ include "resource.default.namespace" . }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "labels.selector" . | nindent 6 }}
+  replicas: 1
+  revisionHistoryLimit: 3
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        {{- include "labels.selector" . | nindent 8 }}
+      annotations:
+        releasetime: {{ $.Release.Time }}
```

```suggestion
        releasetime: {{ $.Release.Revision }}
```

We need this for Helm 3.

---

I will rename the field to `releaseRevision`